src/plugins/typechecker/typechecker/simplespecification.ini

```diff
@@ -0,0 +1,26 @@
+#@META elektra/spec/type = RegexContains b a => Key b :: . -> Key a -> Key a
```

Comment:
It is unclear what the user needs to write.

Reply:
i hope its a bit more clear now. The idea is that such specifications are not done by the user but by the plugin authors. Users shouldn't have to deal with that unless they have some very special cases. In any way i have altered the way this is handled by adding a prelude file containing common definitions like for links or our basic checks so that doesn't have to be written over and over.